releasetasks/templates/source.yml.tmpl

```diff
@@ -72,3 +73,32 @@
             build:
                 platform: none
         #}
+-
+    taskId: "{{ stableSlugId(buildername_signing) }}"
```

Comment:
This Task looks very generic to me, I'm wondering if it should be its own template, so that it can be used by including by other builders that want to sign things too? What do you think?